### PR TITLE
Embed rusk-recovery binary

### LINKED ISSUE #1126: Add `rusk-recovery` capability to `rusk`
#### Summary
Currently, `rusk-recovery` binaries are required in order to recovery the keys and the initial genesis state.
This features shall be available to the `rusk` binary

#### Possible solution design or implementation
Having multiple subcommands would be helpful.
E.g: `./rusk recovery-keys` or `./rusk recovery-state`

#### Additional context
N/A

### PR DESCRIPTION
**Pull request by herr-seppia**
~When `recovery` feature is enable~ According to the enabled features, the following subcommand are available

- `recovery-state`: equivalent to `rusk-recovery-state` binary
- `recovery-keys`: equivalent to `rusk-recovery-keys` binary

Resolves #1126

### CODE REVIEW

File: rusk/src/bin/args/command.rs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            }
+        };
+
+        if let Err(e) = &result {
+            tracing::error!("{} {e}", theme.error("Error"));
+        }
+
+        let finished = theme.action("Finished");
+        let elapsed = begin.elapsed().as_secs_f32();
+        info!("{finished} task in {elapsed:.2}s",);
```
Comment (ureeves) [inline rusk/src/bin/args/command.rs:81]: Not sure if I like measuring how long it takes in the tool itself. There's more than enough measuring tools externally. Any particular reason why we *always* want this?

Comment (herr-seppia) [reply rusk/src/bin/args/command.rs:81]: It was part of rusk-recovery. Maybe it's a good change to get rid of it

Comment (ureeves) [reply rusk/src/bin/args/command.rs]: Oh, if this was the case then don't bother changing it.

**Review by ureeves — APPROVED** [review]
LGTM

File: rusk/src/bin/args/command.rs (hunk 1 above)
Comment (herr-seppia) [reply rusk/src/bin/args/command.rs]: Too late

**Pull request merged**